prime-router/src/main/kotlin/fhirengine/translation/hl7/schema/ConfigSchema.kt

```diff
@@ -161,6 +161,10 @@ abstract class ConfigSchemaElement(
 ) {
     private var validationErrors: MutableSet<String> = mutableSetOf()
 
+    override fun toString(): String {
+        return "${if (schema != null) "$schema-" else ""}$name"
```

Comment:
This could be a bit more readable by extracting out the condition from the string template.